app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
I like attributes better than facets, so it is Ok if the clarification is needed (i.e. subscription vs subscription_attributes).  We just had not used it on any other 'child' nodes, so I wasn't sure if we were trying to add a convention or comply with one.